app/src/main/cpp/skyline/gpu/texture/texture.cpp

```diff
@@ -272,7 +272,7 @@ namespace skyline::gpu {
           layerCount(guest->layerCount),
           sampleCount(vk::SampleCountFlagBits::e1) {
         vk::ImageUsageFlags usage{vk::ImageUsageFlagBits::eTransferSrc | vk::ImageUsageFlagBits::eTransferDst | vk::ImageUsageFlagBits::eSampled};
-        if (format->vkAspect & vk::ImageAspectFlagBits::eColor)
+        if (format->vkAspect & vk::ImageAspectFlagBits::eColor && format->blockWidth == 1 && format->blockHeight == 1)
```

Comment:
Would be optimal if a `format->IsCompressed()` function was added to do this check in a more explicit manner.